cli/tools/check.rs

```diff
@@ -503,23 +546,27 @@ fn get_tsc_roots(
   for specifier in get_import_specifiers() {
     let specifier = graph.resolve(specifier);
     if seen.insert(specifier) {
-      pending.push_back((specifier, false));
+      pending.push_back((specifier, false, true));
     }
   }
```

Comment:
Instead of keeping track of `is_import_specifier`, maybe we should just include the resolved npm specifier -> file url in the roots?